xrpl4j-address-codec/src/test/java/org/xrpl/xrpl4j/codec/addresses/XAddressTest.java

```diff
@@ -246,7 +233,7 @@ public void xAddressWithBadPrefix() {
   public void xAddressWith64BitTag() {
     XAddress xAddress = XAddress.of("XVLhHMPHU98es4dbozjVtdWzVrDjtV18pX8zeUygYrCgrPh");
 
-    assertThrows(
+    org.junit.jupiter.api.Assertions.assertThrows(
```

Comment:
Plz revert this change.

```suggestion
    assertThrows(
```